website/source/docs/providers/google/r/compute_snapshot.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
We're going to need to add this resource to the menu as well, but I can always fix that up after.